fe/fe-core/src/main/java/com/starrocks/http/rest/SyncCloudTableMetaAction.java

```diff
@@ -0,0 +1,86 @@
+
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package com.starrocks.http.rest;
+
+import com.google.common.base.Strings;
+import com.starrocks.common.DdlException;
+import com.starrocks.common.StarRocksHttpException;
+import com.starrocks.http.ActionController;
+import com.starrocks.http.BaseRequest;
+import com.starrocks.http.BaseResponse;
+import com.starrocks.http.IllegalArgException;
+import com.starrocks.lake.StarMgrMetaSyncer;
+import com.starrocks.privilege.AccessDeniedException;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.sql.ast.UserIdentity;
+import io.netty.handler.codec.http.HttpMethod;
+import io.netty.handler.codec.http.HttpResponseStatus;
+
+// This class is responsible for sync cloud table's tablet and shard meta between fe and starmgr
+public class SyncCloudTableMetaAction extends RestBaseAction {
+    public SyncCloudTableMetaAction(ActionController controller) {
+        super(controller);
+    }
+
+    public static void registerAction(ActionController controller)
+            throws IllegalArgException {
+        SyncCloudTableMetaAction action = new SyncCloudTableMetaAction(controller);
+        controller.registerHandler(HttpMethod.GET,
+                "/api/{" + DB_KEY + "}/{" + TABLE_KEY + "}/_sync_meta", action);
+    }
+
+    @Override
+    public void executeWithoutPassword(BaseRequest request, BaseResponse response)
+            throws DdlException, AccessDeniedException {
+        UserIdentity currentUser = ConnectContext.get().getCurrentUserIdentity();
+        checkUserOwnsAdminRole(currentUser);
+
+        if (redirectToLeader(request, response)) {
+            return;
+        }
+
+        String dbName = request.getSingleParameter(DB_KEY);
+        String tableName = request.getSingleParameter(TABLE_KEY);
+        if (Strings.isNullOrEmpty(dbName) || Strings.isNullOrEmpty(tableName)) {
+            throw new StarRocksHttpException(HttpResponseStatus.BAD_REQUEST,
+                    "both database and table should be provided.");
+        }
+
+        StarMgrMetaSyncer.syncTableMeta(dbName, tableName);
```

Comment:
will it be a HTTP internal error if this line excepted?

Reply:
it will be handled by outer layer correctly